### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by TanmayHiremath**
Created the gettingstarted.md file in the /docs folder which includes steps for installation on linux, removed manual approval from django admin after each test, added apis for downloading report and result files

### CODE REVIEW

File: rest/serializers/test_webhook.py [hunk 1]
```diff
@@ -31,7 +32,17 @@ def validate(self, attrs):
                 self.test.negative = attrs.get('negative')
                 self.test.inconclusive = attrs.get('inconclusive')
                 self.test.report_filename =attrs.get('report_filename')
-                self.test.status = Status.objects.get(pk=2)
+                self.test.status = Status.objects.get(name='COMPLETED')
```
Comment (vyasakanksha) [inline rest/serializers/test_webhook.py:35]: did you test and make sure that this works?

**Comment by vyasakanksha** [comment]
1. We also need to send an email to the admin with the results (along with the user)
2. There in an error with the build. Did you install a new dependency or something?

File: rest/serializers/test_webhook.py (hunk 1 above)
Comment (aditakarsh27) [reply rest/serializers/test_webhook.py:35]: It was also being used in https://github.com/Tapestry-Pooling/tapestry-server/blob/954e652c68c617bddd467ba7df2b06f8e20c578c/rest/admin/test.py

**Comment by TanmayHiremath** [comment]
> 1. We also need to send an email to the admin with the results (along with the user)
> 2. There in an error with the build. Did you install a new dependency or something?

1. We just added this in the latest commit
2. It seems that the version of sourcedefender specified in requirements.txt is not available. We haven't installed any dependency.

**Pull request merged**